lib/livebook_web/live/output_component.ex

```diff
@@ -0,0 +1,155 @@
+defmodule LivebookWeb.OutputComponent do
+  use LivebookWeb, :live_component
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div id={"output-#{@id}"}>
+      <%= render_output(@output, %{
+            id: @id,
+            socket: @socket,
+            session_id: @session_id,
+            runtime: @runtime,
+            cell_validity_status: @cell_validity_status,
+            input_values: @input_values
+          }) %>
+    </div>
+    """
+  end
+
+  defp render_output(text, %{id: id}) when is_binary(text) do
+    # Captured output usually has a trailing newline that we can ignore,
+    # because each line is itself an HTML block anyway.
+    text = String.replace_suffix(text, "\n", "")
+    live_component(LivebookWeb.Output.TextComponent, id: id, content: text, follow: true)
```

Comment:
Can you remind me again why we are using stateful components here, if those cells (except frames) are static? :)

Reply:
Previously we wanted to avoid re-rendering when consecutive outputs changed, but this output didn't change. Now that its inside a statefull OutputComponent it's less critical, but we would re-render if input values change (see OutputComponent.render). wdyt?

Reply:
Do you mean if a Kino.Input change? That’s fine to reload I think. The biggest concern with re-rendering is in tight loops, no?

Reply:
Yeah, we do:

```
<%= render_output(@output, %{ ... input_values: @input_values }) %>
```

So if Kino.Input value changes we would call `render_output` and if the text component is stateless then we render it too, even though the content is the same.

But I've just realised that we tailor input_values for every cell, so the impact is limited to the given cell and we should be fine :D I will update the components to stateless.